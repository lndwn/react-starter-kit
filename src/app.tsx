import React from 'react'
import styled from 'styled-components'
import { Flex } from './components'

const Heading = styled.h1`
  color: ${(props) => props.theme.colors.text[2]};
`

interface IAppProps {
  heading?: string
}

export const App = ({ heading = 'Hello World' }: IAppProps) => {
  return (
    <Flex alignItems="center" justifyContent="center">
      <Heading>{heading}</Heading>
    </Flex>
  )
}
