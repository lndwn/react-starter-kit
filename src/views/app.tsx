import * as React from 'react'
import { useFetch } from 'utils/use-fetch'

export const App = () => {
  const thing = useFetch(
    'https://hacker-news.firebaseio.com/v0/item/8863.json?print=pretty',
    {}
  )
  return <>Hello World</>
}
