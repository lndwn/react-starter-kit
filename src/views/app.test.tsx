import * as React from 'react'
import { render } from '@testing-library/react'
import { App } from './app'

describe('root', () => {
  test('renders', () => {
    const { container, getByText } = render(<App />)

    expect(getByText(/hello world/i))
    expect(container.firstChild).toMatchSnapshot()
  })
})
