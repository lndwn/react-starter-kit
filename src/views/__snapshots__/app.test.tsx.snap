// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`root renders 1`] = `Hello World`;
